packages/react-components/src/components/table-output-component.tsx

```diff
@@ -30,9 +30,17 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
     private components: any;
     private gridApi: GridApi | undefined = undefined;
     private columnApi: ColumnApi | undefined = undefined;
-    private pendingIndex: number | undefined = undefined;
-    private lastIndex = { timestamp: Number.MIN_VALUE, index: 0 };
+    private lastSelectedRange = {
+        startTimestamp: Number.MIN_VALUE,
+        endTimestamp: Number.MIN_VALUE,
+        startIndex: 0,
```

Comment:
We don't seem to be using the startIndex/endIndex anywhere?

Reply:
updated.